src/ui/PrimitiveBlock.js

```diff
@@ -1,16 +1,11 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import {ASTNode} from '../ast';
-import Renderer from '../Renderer';
 import {Primitive} from '../parsers/primitives';
-
-require('./PrimitiveBlock.less');
+import {RendererContext} from './Context';
+import './PrimitiveBlock.less';
```

Comment:
Maybe this is a stupid question, but aren't the semantics of `import` such that this could be compiled without `PrimitiveBlock.less`? Wouldn't that be problematic?